sd_jwt/examples/18013_5.yml

```diff
@@ -0,0 +1,46 @@
+only copy-paste, needs to be updated:

```

Comment:
Is this intended to be here?

Reply:
the same by me, shouldn't this text be removed?